assets/sass/_content-post.scss

```diff
@@ -17,6 +17,17 @@
     text-transform: none;
   }
 
+  a.external::after {
```

Comment:
So, you'd like to see this for *every* external link (even the ones to github  docs)?  I was thinking of being selective about it, say for only ABE and AMP Start.